invenio_app_ils/document_requests/validator.py

```diff
@@ -25,28 +25,28 @@ def validate_state(self, state, valid_states):
     def validate_document_pid(self, document_pid, state, reject_reason):
         """Validate data for accepted state."""
         # Accepted requests must have a document and a provider (ILL, ACQ)
-        if state == "ACCEPTED":
-            if document_pid:
-                try:
-                    search = DocumentRequestSearch()
-                    count = search.search_by_document_pid(document_pid).count()
-                    if count > 0:
-                        raise DocumentRequestError(
-                            "Document PID {} already has a request".format(
-                                document_pid
-                            )
-                        )
-                except PIDDoesNotExistError:
-                    # Invalid document_pid
+        if document_pid:
+            try:
+                search = DocumentRequestSearch()
+                count = search.search_by_document_pid(document_pid).count()
+                if count > 0:
                     raise DocumentRequestError(
-                        "State cannot be ACCEPTED because a document with "
-                        "PID {} doesn't exist".format(document_pid)
+                        "Document PID {} already has a request".format(
```

Comment:
NOTHING TO DO
there might be many document request per one document. Especially when f.e. there is not enough copies or there is many rejections of type "found in catalogue" . I know that you just reorganised the code, I'm just referencing it here

Reply:
Yes, this is most likely a left-over since when we only allowed one.